docs/Relations.md

```diff
@@ -0,0 +1,36 @@
+
+## The `Relations` module {#sec-relations}
+
+The `Relations` module defines a number of properties of relations, which are predicates of two arguments. In particular, this module introduces properties of functions, which can be understood as relations of a particular type.
+
+Properties of binary predicates:
+- Reflexive
+- Irreflexive
+- AntiSymmetric
+- Transitive
+- Connected
+- StronglyConnected
+- TotalOrdering
+- StrictTotalOrdering
```

Comment:
This is wonderful.
But it would also be great if we had a few words on each.
Or a link to some more details :)

Reply:
I added some (mathematical detail)

Reply:
Have you pushed that yet? I don't see it, and I agree it'd be lovely to have.